protocol.md

```diff
@@ -103,6 +103,31 @@ To verify:
 Either standard or URL-safe base64 encodings are allowed. Signers may use
 either, and verifiers **MUST** accept either.
 
+## Multi-signature Verification
+
+Multi-signature enhances the security by allowing multiple signers to sign the
+same payload. The resulting signatures are encoded and transmitted, preferably
+using the recommended [JSON envelope](envelope.md).
+
+A `(t, n)`-ENVELOPE is valid if the enclosed signatures pass the verification
+against at least `t` of `n` unique trusted public keys.
+
+To verify a `(t, n)`-ENVELOPE:
+
+-   Receive and decode SERIALIZED_BODY, PAYLOAD_TYPE, SIGNATURES from ENVELOPE.
+    Reject if decoding fails.
+-   For each (SIGNATURE, KEYID) in SIGNATURES,
```

Comment:
You may not always get the keyid...

Reply:
KEYID can be `nil`. I won't fix it here since it need to be consistent with the above protocol for single signatures.

Reply:
Got it. Maybe we just need to say something like, "Be aware that KEYID may be `nil`."

Reply:
The KEYID is defined in the section [Signature Definition](https://github.com/secure-systems-lab/dsse/blob/master/protocol.md#signature-definition) and is clearly marked as `Required = No` (i.e. optional). I don't think we need to redundantly mention this everywhere.